.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: required
 language: php
 
 php:
-  - 5.6
+  - 7.3
```

Comment:
Will this impact compatibility with merchants that are currently using our WooCommerce gateway with PHP <7.3 versions?

Reply:
Those who are using the plugin NOW will not be impacted. But by now, all merchants who want to use this plugin will have to met the plugin requirements; those requirements are the same as the [benjamin](https://github.com/ebanx/benjamin).